huxley/core/models.py

```diff
@@ -292,6 +294,9 @@ def remove(assignment_data):
             Assignment.objects.filter(id__in=deletions).delete()
             Assignment.objects.bulk_create(additions)
 
+    def finalize(self):
```

Comment:
This method doesn't look like it's used any more. It's even referencing an outdated field.